route4me-csharp-sdk/Route4MeSDKLibrary/Route4MeManagerV5.cs

```diff
@@ -1293,13 +1326,17 @@ public RouteDuplicateResponse DuplicateRoute(string[] routeIDs, out ResultRespon
 
             var genParams = new RouteParametersQuery();
 
-            return GetJsonObjectFromAPIAsync<RouteDuplicateResponse>(
+            var result = GetJsonObjectFromAPIAsync<RouteDuplicateResponse>(
                 genParams,
                 R4MEInfrastructureSettingsV5.RoutesDuplicate,
                 HttpMethodType.Post,
                 content,
                 false,
                 false);
+
+            return Task.FromResult(
```

Comment:
What is the purpose of this change? I would recommend to avoid unnecessary memory traffic here. You can just return result or revert changes.

Reply:
See the comment above for similar question.